vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/core_schema.go

```diff
@@ -285,6 +288,25 @@ func (s *Schema) coreConfigSchemaType() cty.Type {
 	}
 }
 
+func (r *Resource) ProtoSchema(ctx context.Context) *tfprotov5.Schema {
```

Comment:
These were taken from https://github.com/hashicorp/terraform-plugin-sdk/tree/av/schema-method and may be introduced for actions.